lib/chef/mixin/user_context.rb

```diff
@@ -0,0 +1,52 @@
+#
+# Author:: Adam Edwards (<adamed@chef.io>)
```

Comment:
If you can get a branch with it rebased and all merge conflicts resolved, I can get the tests passing again...